utils/vale/styles/Crossplane/spelling-exceptions.txt

```diff
@@ -20,16 +21,19 @@ minikube
 namespace
 namespaces
 Netlify
+Postgre
```

Comment:
is this correct? from what i've seen, its typically referred to as `PostgreSQL` or `postgres`